zeppelin-server/notebook/python.recovery

```diff
@@ -1 +0,0 @@
-2CZA1DVUG:shared_process	192.168.3.2:55410
```

Comment:
is this intentional, python.recovery and python.recovery.crc file?

Reply:
yes, these files are not used, should be removed.